src/main/common/time.c

```diff
@@ -51,6 +52,7 @@ PG_REGISTER_WITH_RESET_TEMPLATE(timeConfig_t, timeConfig, PG_TIME_CONFIG, 0);
 
```

Comment:
You're adding fields to the parameter group. Please also increase version number in the declaration `PG_REGISTER_WITH_RESET_TEMPLATE`

Reply:
Rookie mistake, my bad.

Reply:
No worries, I keep forgetting to do this myself 😉

Reply:
Is there any more work left to be done on this? Sorry to be bothering, it seems that I cannot manually mark this request as solved.